tests/unit_tests/steps/test_root_inc_files.py

```diff
@@ -22,7 +22,7 @@ class TestRootIncFiles:
     """
     Tests include files are handled correctly.
     """
-    def test_vanilla(self, tmp_path: Path) -> None:
+    def test_vanilla(self, tmp_path: Path, recwarn) -> None:
```

Comment:
Just checked the documentation and the type is `recwarn: WarningsRecord`.

Reply:
Thanks, I've updated the type to WarningsRecorder (and added a few more). Strange, I thought `mypy` would warn if there are types missing in a function that has at least some typing applied.